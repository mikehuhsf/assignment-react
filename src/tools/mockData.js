export const platforms ={
    PS5: "NO",
    XBOX_Series_X: "NO",
    Windows: "YES",
    macOS: "NO",
    Linux: "NO",
    Nintendo_Switch: "NO"
}
