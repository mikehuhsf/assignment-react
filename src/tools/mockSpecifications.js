export const specifications ={
    manufacturer: "Apple",
    storage_capacity: "128 GB",
    GPU: "A1",
    CPU: "A1",
}
