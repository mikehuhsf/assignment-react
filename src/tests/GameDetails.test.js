import React from 'react';
import renderer from 'react-test-renderer';
import GameDetails from '../components/GameDetails';
import { platforms } from '../tools/mockData';


it("display's platform support when it exists", () => {

    const wrapper = renderer.crate(<GameDetails release_date='010101' platforms={platforms} />)
    expect(wrapper).toMatchSnapshot();
    
})