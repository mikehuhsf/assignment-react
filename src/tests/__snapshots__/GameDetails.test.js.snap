// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display's platform support when it exists 1`] = `
Array [
  <h5>
    Release Date: 
    010101
  </h5>,
  <h5>
    Platforms: 
    <br />
    <blockquote>
       PS5: 
      NO
       
      <br />
      XBOX Series X: 
      NO
       
      <br />
      Nintendo Switch: 
      NO
       
      <br />
      PC (Windows): 
      YES
      <br />
      PC (macOS): 
      NO
       
      <br />
      PC (Linux): 
      NO
       
    </blockquote>
  </h5>,
]
`;
